src/modules/ID/index.ts

```diff
@@ -1,19 +1,44 @@
-export function getCollectionId(id: string): string | null {
-  let values = id.split(':')
+/**
+ * Gets from a blockchain item id, the collection id.
+ * The blockchain item id is formatted as collectionId:itemId.
+ *
+ * @param collectionAndItemIds - The blockchain id for the item, containing the item id and the collection id.
+ */
+export function getCollectionId(collectionAndItemIds: string): string | null {
```

Comment:
I would have a module for `ThirdParty` and another one for `Item`. In my opinion will be better scoped rather to have a utils with a lot of way of getting ids